frontend/src/shared/Data/AppleWatch/index.ts

```diff
@@ -0,0 +1,51 @@
+import axios, { AxiosPromise } from "axios";
+
+const DEV =
+  "http://138.197.134.112:8080/BeaplabEngine-0.0.1-SNAPSHOT/rest/beapengine/applewatch";
+// const PROD = "/api/rest/beapengine/applewatch";
+
+const $axios = axios.create({
+  baseURL: DEV,
+  withCredentials: true,
+  responseType: "json",
+});
+
```

Comment:
Instead of using this import api from baseapi.ts that lives in `shared/api/baseapi.ts`. It is the equivalent of thuis but baseurl is configured to work with ours automatically based on the environment

I also suggest combining the Fitbit and Applewatch api calls as its the exact same thing. The only thing being different is the api route with one being applewatch and one being fitbit. This will also make it easier if we decide to add garmin as well